components/menu/menu.js

```diff
@@ -259,17 +259,18 @@ class Menu extends HierarchicalViewMixin(LitElement) {
 
 	}
 
-	_onKeyPress(e) {
+	_onKeyUp(e) {
 		if (this._items.indexOf(e.composedPath()[0]) === -1) return;
 
-		if (e.keyCode === keyCodes.DOWN || e.keyCode === keyCodes.UP
-			|| e.keyCode === keyCodes.SPACE || e.keyCode === keyCodes.ENTER
+		if (e.keyCode === keyCodes.SPACE || e.keyCode === keyCodes.ENTER
 			|| e.keyCode === keyCodes.ESCAPE) {
 			return;
 		}
 
 		e.stopPropagation();
 
+		if (e.keyCode === keyCodes.DOWN || e.keyCode === keyCodes.UP) return;
```

Comment:
We don't want these escaping the menu do we?

Reply:
What do you mean by "escaping the menu"?  What behavior does this change/fix?

Reply:
Right now, when you press up and down in a drop down while in an aria-grid. Such as lists the keys leak out to the list and move focus away from the drop down and into the list item.

Reply:
I see.  That's a little concerning.  We can,'t always expect components to stop propagation for that scenario.  Like we can do it here for this component, but what else might we have to deal with?  These events naturally bubble and usually it would be the responsibility of the handlers higher up to check target, etc.

Reply:
That is a solution. How would the list know what target uses what keys though?

Reply:
The only alternative that I can think of is that the list-grid would have to be aware of which types of elements (including checking aria role) to ignore.  Also not amazing.

I think in this case we can probably safely stop the propagation, since the menu is providing some very specific behavior and it's unlikely that parent components would be interested in those events.  I wouldn't say that about all events though, or all key codes, which is what makes me pause here.